docs/reference/filters/pluralize.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Pluralize
+parent: Filters
+grand_parent: Reference
+---
+
+# pluralize
+
+Outputs the singular or plural version of a string based on a given number.
+
+### Example:
+
+{% raw %}
+```liquid
+<h1>{{ item_count }} {{ item_count | pluralize: 'item' }} in your cart</h1>
+```
+{% endraw %}
+
+The above would result in `<h1>2 items in your cart</h1>` when `item_count` has the value of `2`.
+
+This filter accepts a custom plural version as a param.
+
+### Example:
+
+{% raw %}
+```liquid
+<h1>{{ people_count }} {{ people_count | pluralize: 'person', 'people' }} in your venue</h1>
```

Comment:
I think "people" is the pluralised version of "person" that Rails' pluralize would return.

Can we use "users" in this example instead?

```
{{ people_count | pluralize: 'person', 'users' }}
```

Reply:
I can, I feel like rails internals aren't very relevant for the given example thought.

Reply:
I just mean that our `pluralize` filter will already use the plural string of "people". So this example is a bit redundant. 

It would be better to use a _different_ plural string than what the filter would normally return.

Reply:
No worries, have updated, definitely don't want to mislead the developers to pluralise it on that way when that's already what happened.